### PR TITLE
Chore/update linters to new standard

### PR DESCRIPTION
**Pull request by jaguirresza**
This PR updates linters used for node apps. I will need help from @rodrwan for the scrapers' linter and from both, @cagodoy and @rodrwan, for the front end linter.

Also, add a `.rubocop.yml` for ruby projects.

### CODE REVIEW

File: javascript/Readme.md [hunk 1]
```diff
@@ -1,5 +1,159 @@
...
+});
+
+// good
+[1, 2, 3].map(x => {
+  const y = x + 1;
+  return x * y;
+});
+```
+
+- We use special rultes for using the `let` variable declaration. All variables declared with let must be declared at the beginning of the scope.
```
Comment (rodrwan) [inline javascript/Readme.md]: typo, should said: `spetial rules`:trollface:, also `let`should be mark as special variable with '`' provided by markdown

**Comment by rodrwan** [comment]
good job :rocket:

File: javascript/Readme.md (hunk 1 above)
Comment (jaguirresza) [inline javascript/Readme.md]: You are right, thanks. Fixed in a623de2

**Comment by sebastianvera** [comment]
:tada: :birthday: :balloon:

**Comment by cagodoy** [comment]
yeah @jorgeaguirre, great job :D :cactus: :monkey:

**Pull request merged**